modules/cli-developer-other.adoc

```diff
@@ -30,9 +30,15 @@ $ oc plugin list
 
 == version
 
-Display the `oc` client and server versions.
+Display the `oc` client and Kubernetes server versions.
 
 .Example: Display version information
 ----
 $ oc version
 ----
+
+ifdef::openshift-enterprise,openshift-origin[]
+
+For cluster administrators, the OpenShift server version is also displayed.
```

Comment:
s/OpenShift/{product-title} always and forever. We get bugs whenever "OpenShift" appears in the OKD collection. :/

Reply:
Good call!